pkg/webhook/parser.go

```diff
@@ -362,20 +404,54 @@ func (Generator) RegisterMarkers(into *markers.Registry) error {
 	if err := into.Register(ConfigDefinition); err != nil {
 		return err
 	}
+	if err := into.Register(WebhookConfigDefinition); err != nil {
+		return err
+	}
 	into.AddHelp(ConfigDefinition, Config{}.Help())
+	into.AddHelp(WebhookConfigDefinition, Config{}.Help())
 	return nil
 }
 
 func (g Generator) Generate(ctx *genall.GenerationContext) error {
 	supportedWebhookVersions := supportedWebhookVersions()
 	mutatingCfgs := make(map[string][]admissionregv1.MutatingWebhook, len(supportedWebhookVersions))
 	validatingCfgs := make(map[string][]admissionregv1.ValidatingWebhook, len(supportedWebhookVersions))
+	mutatingWebhookCfgs := make(map[string]admissionregv1.MutatingWebhookConfiguration, len(supportedWebhookVersions))
```

Comment:
map where values are the object instead of array or objects since there's only one MutatingWebhookConfiguration and one ValidatingWebhookConfiguration per supported version